### ISSUE TITLE
Scrolling in xfce terminal is broken

### ISSUE THREAD

**Issue by vladimir-malky** [randy3k/rtichoke#44]
Hi, I wanted to let you know that scrolling in xfce4 terminal doesn't work. If there's output larger than window and I want to scroll up, it immediately resets back to active prompt. I tested KDE konsole also, where it works as expected. Just wanted to let you know, anyway thanks for great R repl.

**Comment by randy3k** [comment]
Could you briefly describe how to reproduce it? I cannot reproduce it when opening `xfce4-terminal` via ssh.

**Comment by vladimir-malky** [comment]
Hi, sorry I didn't get back to you sooner. I just reproduced it via VNC from my phone. Not sure how to test GUI through SSH. Probably easiest way to reproduce would be to install xubuntu or some other distro in virtual machine, if you don't have linux desktop.

Steps to reproduce:
1) Open xfce4-terminal in some linux desktop environment
2) Create output larger than window, for example large number sequence like 1:100
3) Try to scroll to beginning of output

Expected behaviour: you are able to scroll up to see beginning of output or previous commands.

Observed behaviour: in the moment I stop scrolling it snaps back to active prompt, blocking me from viewing previous output.

I don't think it's extremely important, just kind of annoying. I just wanted to let you know, if you feel like diving in and investigate :)

**Comment by randy3k** [comment]
I have actually tried installing xfce4-terminal on Ubuntu virtual machines (both Kubuntu and Gnome Ubuntu) and still couldn't reproduce it.

I even tried installing xfce desktop to a remote server and used vnc to login. And I also wasn't able to reproduce it.

**Comment by beazlr02** [comment]
I have this on xfce4-terminal 0.8.3 (Xfce 4.12)

**Comment by randy3k** [comment]
I have xfce4-terminal 0.6.3 on gnome ubuntu and cannot reproduce it. Let me try again with xfce desktop.
